src/app/features/comics/comics.component.ts

```diff
@@ -0,0 +1,46 @@
+
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { BannerComponent } from '@core/components/banner/banner.component';
+import { ComicCardComponent } from '@comics/components/comic-card/comic-card.component';
+import { CustomMarvelComic } from '@comics/models/comics.model';
+import { ComicsMarvelService } from '@comics/services/comics-marvel.service';
+
+@Component({
+  selector: 'app-comics',
+  standalone: true,
+  imports: [ ComicCardComponent, BannerComponent],
+  templateUrl: './comics.component.html',
+  styleUrl: './comics.component.scss'
+})
+export class ComicsComponent {
+  comicsList: CustomMarvelComic[] = [];
+  amountOfReceivedComics: number = 0;
+
+  comicsLimit: number = 15;
+  comicsOffset: number = 150;
```

Comment:
why 150?

Reply:
I just want to see different comics, so I just change the number sometimes)